conf.py

```diff
@@ -161,16 +161,21 @@
         ),
         (
           (
+            ("/mission-statement/", "Our mission statement", ""),
             ("/how-ansible-works/", "How Ansible works", ""),
             ("/ecosystem/", "Ansible ecosystem", ""),
+            ("/awx/", "AWX", ""),
+            ("/galaxy/", "Galaxy", ""),
+            ("/faq/", "FAQs", ""),
             ("/ansible-community-training/", "Ansible community training", ""),
-            ("/mission-statement/", "Our mission statement", ""),
+            
             ("/contact-us/", "Contact us", ""),
           ),
           "Resources", ""
         ),
-        ("https://forum.ansible.com/", "Ansible community forum", ""),
+        ("https://forum.ansible.com/", "Forum", ""),
         ("https://docs.ansible.com/", "Documentation", ""),
+        ("https://www.redhat.com/en/technologies/management/ansible/", "Ansible Automation Platform", ""),
```

Comment:
Should this be "Red Hat Ansible Automation Platform"?

Reply:
No, the current label was approved by ABU Marketing.